jobs/delorean/2.0/ews/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+
+pipeline {
+    agent none
+
+    parameters {
+        // Example message:
+        // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-43656-1547130290231-6:586552:0:0:1&is_raw=true&size=extra-large
+        string defaultValue: '', description: '', name: 'CI_MESSAGE', trim: true
+    }
+
+    options {
+        disableConcurrentBuilds()
+        buildDiscarder logRotator(numToKeepStr: '3', daysToKeepStr: '8')
```

Comment:
You might not need/want to set this -- I was just doing it on the other Jenkins because it had very limited resources & space, etc.

Reply:
I've removed those for now. @mikenairn if there's any options we want to pass here I can add those back in later.